infrastructure/running-a-node/setup-full-node.md

```diff
@@ -1,4 +1,355 @@
 ---
-title: Set Up a Full Node
-description: TODO
----
\ No newline at end of file
+title: Set Up a Node
+description: Instructions on how to set up a node in accordance to your platform and the differences between different node types.
```

Comment:
```suggestion
description: Learn how to install, configure, and run Polkadot nodes, including setting up different node types and connecting to the network.
```

While working on index pages, saw that this was a little short. I'll review the rest of this PR tomorrow AM